src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard list App</title>
+</head>
+<body>
+  <main>
+    <div class="main-cont">
+      <h1 class="leaderboard-title">Leaderboard</h1>
+      <section class="container">
+        <div class="main-sec">
+          <div class="first-part">
+            <h2 class="recent-score">Recent Scores</h2>
+            <button class="refresh">Refresh</button>
+          </div>
+          <ul class="score-list">
+            
+          </ul>
```

Comment:
- [ ] While elaborate styling is not a requirement for this project you are still required to maintain the layout of the elements as we have in the wireframe. Kindly add some dummy scores to your app for demo purposes and give an illustration of the position of the scores.

- [ ] Kindly move the refresh button to the right side of the scores section to match the design.

  Your layout | Wireframe layout
  :---:|:---:
  ![ieoeww](https://user-images.githubusercontent.com/36542986/223665230-e0d21151-4c45-430f-8e89-88c9fa71cf43.jpg) | ![kriw](https://user-images.githubusercontent.com/36542986/223665241-5a9d2a44-1613-465e-9105-d3eca8ea7c25.jpg)